website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -98,4 +98,8 @@ Define configurable data retention policies for workspaces to help reduce object
 
 Once an object is soft deleted, any attempts to read the object will fail. Until the garbage collection grace period elapses, you can still restore an object using the APIs described in the [configuration version documentation](/terraform/enterprise/api-docs/configuration-versions) and [state version documentation](/terraform/enterprise/api-docs/state-versions). After the garbage collection grace period elapses, Terraform permanently deletes the archivist storage.
 
-The [organization policy](/terraform/enterprise/users-teams-organizations/organizations#destruction-and-deletion) is the default policy applied to workspaces, but members of individual workspaces can override the policy for their workspaces. The workspace policy always overrides the organization policy.
+The [organization policy](/terraform/enterprise/users-teams-organizations/organizations#destruction-and-deletion) is the default policy applied to workspaces, but members of individual workspaces can override the policy for their workspaces.
+
+The workspace policy always overrides the organization policy, and there are three types of data retention policies that a workspace admin has the ability to set or override: the organization default policy, do not auto-delete, and auto-delete data.
```

Comment:
```suggestion
The workspace policy always overrides the organization policy. An workspace owner can set or override the following data retention policies: 

- **Organization default policy** 
- **Do not auto-delete** 
- **Auto-delete data**
```
If these policies names do not correspond to a UI label, then they do not need to be in boldface.

Reply:
They do correspond to the labels, so I will leave them boldface!